app/controllers/api/databases.php

```diff
@@ -3023,10 +3032,29 @@ function updateAttribute(
             }
         };
 
-        $checkPermissions($collection, $document, Database::PERMISSION_CREATE);
+        $documents = array_map(function ($document) use ($collection, $permissions, $checkPermissions, $isBulk, $documentId) {
+            $document['$collection'] = $collection->getId();
+            $document['$permissions'] = $permissions;
```

Comment:
We should only do this for single documents, for bulk, $permissions should be inside the documents like $id